frr_python/frr_server.py

```diff
@@ -0,0 +1,72 @@
+"""A Fleetspeak server service
+
+Sends TrafficRequestData messages to a client and receives back
+TrafficResponseData messages
+"""
+
+import binascii
+import logging
+import time
+
+from absl import app
+from absl import flags
+
+from fleetspeak.server_connector.connector import InsecureGRPCServiceClient
+from fleetspeak.src.common.proto.fleetspeak.common_pb2 import Message
+from fleetspeak.src.inttesting.frr.proto.fleetspeak_frr.frr_pb2 import TrafficRequestData
+from fleetspeak.src.inttesting.frr.proto.fleetspeak_frr.frr_pb2 import TrafficResponseData
+
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string(
+    name="client_id",
+    default="",
+    help="An id of the client to send the messages to.")
+
+
+def listener(message, context):
```

Comment:
Though the casing "listener" is OK, we tend to use the casing "Listener" in our Python, code.